client/src/components/EventTypeDialog.js

```diff
@@ -78,7 +78,42 @@ export default function EventTypeDialog() {
 
     const handleColorChange = (color) => {
         setColor('#'+color.hex)
-    }
+    };
+    
+    const handleConfirm = () => {
+        if (title===''||duration===''||url===''||color==='') return alert('Missing Field(s)') //might need to change after discuss
+        else {
+            let status;
+            fetch("/event-types/", {
+                method: "POST",
+                headers: {
+                    "Content-Type": "application/json"
+                },
+                body: JSON.stringify({
+                    // need to decide get 'user_id' from cookie by BE or store in state
```

Comment:
on the frontend user_id should be stored in context (pretty much just state at a high level). On the backend, the middleware can get the user_id from the JWT